runtime/oti/j9javaaccessflags.h

```diff
@@ -30,112 +30,133 @@
 
 /* @ddr_namespace: map_to_type=J9JavaAccessFlags */
 
-/* Constants from J9JavaAccessFlags */
-#define J9AccAbstract 0x400
-#define J9AccAnnotation 0x2000
-#define J9AccBridge 0x40
+/*
+ * Modifiers of ROM classes, fields or methods.
+ * When a given modifier is defined by the JVM specification,
+ * the value is taken from that specification.
+ *
+ * See map in ROMClassWriter::writeMethods to confirm available slots for methods.
+ */
+#define J9AccPublic 0x1             					/* class method field */
+#define J9AccPrivate 0x2            					/* class(nested) method field */
+#define J9AccProtected 0x4          					/* class(nested) method field */
+#define J9AccStatic 0x8             					/* class(nested) method field */
+#define J9AccFinal 0x10             					/* class method field */
+/* Project Valhalla repurposes 0x20 as ACC_IDENTITY since ACC_SUPER has no effect after Java 8. */
+#define J9AccSuper 0x20             					/* class */
+#define J9AccClassHasIdentity 0x20  					/* class */
+#define J9AccSynchronized 0x20							/* method */
+#define J9AccValueType 0x40 							/* class(Valhalla) */
+#define J9AccBridge 0x40 								/* method */
+#define J9AccVolatile 0x40          					/* field */
+#define J9AccVarArgs 0x80 								/* method */
+#define J9AccTransient 0x80         					/* field */
+#define J9AccNative 0x100 								/* method */
+#define J9AccInterface 0x200        					/* class */
+#define J9AccGetterMethod 0x200 						/* method */
+#define J9AccAbstract 0x400 							/* class method */
```

Comment:
Please use tabs only for indentation, not after any non-tab character.
```c
#define J9AccPublic                               0x1 /* class method field */
#define J9AccPrivate                              0x2 /* class(nested) method field */
#define J9AccProtected                            0x4 /* class(nested) method field */
#define J9AccStatic                               0x8 /* class(nested) method field */
#define J9AccFinal                               0x10 /* class method field */
/* Project Valhalla repurposes 0x20 as ACC_IDENTITY since ACC_SUPER has no effect after Java 8. */
#define J9AccSuper                               0x20 /* class */
#define J9AccClassHasIdentity                    0x20 /* class */
#define J9AccSynchronized                        0x20 /* method */
#define J9AccValueType                           0x40 /* class(Valhalla) */
#define J9AccBridge                              0x40 /* method */
#define J9AccVolatile                            0x40 /* field */
#define J9AccVarArgs                             0x80 /* method */
#define J9AccTransient                           0x80 /* field */
#define J9AccNative                             0x100 /* method */
#define J9AccInterface                          0x200 /* class */
#define J9AccGetterMethod                       0x200 /* method */
#define J9AccAbstract                           0x400 /* class method */
```
I'll let you decide whether the constants should all be the same width:
```c
#define J9AccPublic                        0x00000001 /* class method field */
#define J9AccPrivate                       0x00000002 /* class(nested) method field */
#define J9AccProtected                     0x00000004 /* class(nested) method field */
#define J9AccStatic                        0x00000008 /* class(nested) method field */
#define J9AccFinal                         0x00000010 /* class method field */
/* Project Valhalla repurposes 0x000000020 as ACC_IDENTITY since ACC_SUPER has no effect after Java 8. */
#define J9AccSuper                         0x00000020 /* class */
#define J9AccClassHasIdentity              0x00000020 /* class */
#define J9AccSynchronized                  0x00000020 /* method */
#define J9AccValueType                     0x00000040 /* class(Valhalla) */
#define J9AccBridge                        0x00000040 /* method */
#define J9AccVolatile                      0x00000040 /* field */
#define J9AccVarArgs                       0x00000080 /* method */
#define J9AccTransient                     0x00000080 /* field */
#define J9AccNative                        0x00000100 /* method */
#define J9AccInterface                     0x00000200 /* class */
#define J9AccGetterMethod                  0x00000200 /* method */
#define J9AccAbstract                      0x00000400 /* class method */
```

Reply:
k it should be fixed now.

Reply:
> Please use tabs only for indentation

That part of my comment has not been addressed (you might use `expand -4` to fix them).